NEWS.md

```diff
@@ -52,8 +67,11 @@
 * Fix to prevent sending refresh complete notification if snap snap-refresh-observe interface is connected
 * Fix to queue snapctl service commands if run from the default-configure hook to ensure they get up-to-date config values
 * Fix stop service failure when the service is not actually running anymore
+* Fix parsing /proc/PID/mounts with spaces
 * Add registry interface that provides snaps access to a particular registry view
-* steam-support interface: relaxed AppArmor and seccomp restrictions to improve user experience
+* Add snap-interfaces-requests-control interface to enable prompting client snaps
+* steam-support interface: remove all AppArmor and seccomp restrictions to improve user experience
+* opengl interface: improve compatibility with nvidia drivers
```

Comment:
Was one of these emergency reverted in the past few days?

Reply:
steam support interface was reverted multiple times, but attempt 3 made it through :)

Reply:
I think this is what I'm thinking of, related to lxd-support: 2cbcac583d3ab0949bd6426be3e68fa628f6d60c

Which reverts 4a0c15d0527f559ec617aac5b100a82d12f313d7